src/PHPCR/Util/Console/Command/WorkspaceImportCommand.php

```diff
@@ -38,6 +46,15 @@ protected function configure()
 
 If the <info>parentpath</info> option is set, the document is imported to that
 path. Otherwise the document is imported at the repository root.
+
+The optional <info>uuid-behavior</info> option describes how uuids should be
+handled. Following options are available:
+
+* <info>new</info> recreate the uuid foreach node
+* <info>remove</info> remove the node from imported dataset on uuid collision
```

Comment:
just fyi: remove does remove the existing node on collision, not the one being imported, according to the jcr specification. i changed the doc accordingly.